SocialMediaDB.sql

```diff
@@ -64,3 +64,87 @@ CREATE TABLE PROFILE (
     sexes ENUM('Male', 'Female', 'Others', 'PreferNotToSay'),
     CONSTRAINT OWNS_PROFILE FOREIGN KEY (user_id) REFERENCES USER (user_id) on DELETE CASCADE on UPDATE CASCADE
 ) ENGINE = InnoDB DEFAULT CHARSET = utf8;
+
+CREATE TABLE EDUCATION (
+    user_id INT,
+    education VARCHAR(100),
+    PRIMARY KEY (user_id, education),
+    FOREIGN KEY (user_id) REFERENCES PROFILE(user_id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE PAGE (
+    page_id int AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    page_name varchar(50) NOT NULL,
+    owner_id int,
+    FOREIGN KEY (owner_id) REFERENCES USER(user_id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE BUSINESS_PLACE (
+    page_id int PRIMARY KEY,
+    owner_name varchar(50) NOT NULL,
+    location varchar(100),
+    FOREIGN KEY (page_id) REFERENCES PAGE(page_id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE PROD_BP(
+    page_id int,
+    prod_name varchar(50),
+    prod_price decimal(10,2),
```

Comment:
```suggestion
    name varchar(50),
    price decimal(10,2),
```
Not important, but better to understand if we follow the naming in the 2NF model.